my-sms/lib/tasks/generate_demo_data.rake

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+namespace :data do
+  desc 'Generate demo data using FactoryBot.'
+  task generate: :environment do
+    require 'factory_bot_rails'
+    80.times { FactoryBot.create(:student, :demo) }
+    30.times { FactoryBot.create(:institute, :demo, with_courses: true) }
```

Comment:
Nice 👍 
Another way to do this is to use factory bot's inbuilt `create_list` method

```ruby
FactoryBot.create_list(:student, 30, :demo)